src/main/java/gift/main/controller/AdminController.java

```diff
@@ -0,0 +1,56 @@
+package gift.main.controller;
+
+import gift.main.dto.ProductRequest;
+import gift.main.entity.Product;
+import gift.main.service.ProductService;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/admin")
+public class AdminController {
+    private final ProductService productService;
+
+    public AdminController(ProductService productService) {
+        this.productService = productService;
+    }
+
+    @GetMapping("/product/list")

```

Comment:
RESTful 하게 list 대신 id 를 적지 않는 것으로 대체할 수 있을 것 같습니다😀
```suggestion
    @GetMapping("/product")

```

fyi; [RESTful API 관련 문서](https://meetup.nhncloud.com/posts/92)

Reply:
넵 제가 이 부분 공부하고 다시 코멘트 달겠습니다!

Reply:
반영했습니다!